example/src/main/java/com/stripe/example/activity/PaymentActivity.java

```diff
@@ -173,4 +178,66 @@ private void initListView() {
                 new int[]{R.id.last4, R.id.tokenId});
         listView.setAdapter(simpleAdapter);
     }
+
+    private void saveCreditCard() {
+        Card cardToSave = createCardToSave();
+
+        if (validateCard(cardToSave)) {
+            startProgress();
+            new Stripe().createToken(
+                    cardToSave,
+                    PUBLISHABLE_KEY,
+                    new TokenCallback() {
+                        public void onSuccess(Token token) {
+                            addToList(token);
+                            finishProgress();
+                        }
+                        public void onError(Exception error) {
+                            handleError(error.getLocalizedMessage());
+                            finishProgress();
+                        }
+                    });
+        }
+    }
+
+    private void saveCreditCardWithRx() {
+        final Card cardToSave = createCardToSave();
+        if (!validateCard(cardToSave)) {
+            return;
+        }
+
+        final Stripe stripe = new Stripe();
+        final Observable<Token> tokenObservable =
+                Observable.fromCallable(
+                        new Callable<Token>() {
+                            @Override
+                            public Token call() throws Exception {
+                                startProgress();
+                                return stripe.createTokenSynchronous(cardToSave, PUBLISHABLE_KEY);
+                            }
+                        });
+
+        compositeSubscription.add(tokenObservable
+                .subscribeOn(Schedulers.io())
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribe(
+                        new Action1<Token>() {
+                            @Override
+                            public void call(Token token) {
+                                addToList(token);
+                                finishProgress();
+                            }
+                        },
+                        new Action1<Throwable>() {
+                            @Override
+                            public void call(Throwable throwable) {
```

Comment:
Bonus: error handling is built into the rx model! No more try/catch! (I tested this by disabling the client-side card verification and pushing invalid data. As a side note, 4242 4242 4242 without the final 4 digits totally reads as a legit card. I get a token back and everything).